src/Device.js

```diff
@@ -15,6 +15,26 @@ class Device extends EventEmitter {
     this.adapter = adapter
     this.device = device
     this.helper = new BusHelper(dbus, 'org.bluez', `/org/bluez/${adapter}/${device}`, 'org.bluez.Device1', { usePropsEvents: true })
+    this.init()
+  }
+
+  async init () {
+    this.helper.on('PropertiesChanged', (propertiesChanged) => {
+      if ('ManufacturerData' in propertiesChanged) {
+        const { value } = propertiesChanged.ManufacturerData
+        if (value) {
+          this.emit('manufacturerData', value)
+        }
+      }
```

Comment:
I understand that you want to proxy the Manufacturer change event to the Device event handler, but currently this isn't available for the rest of props. I prefer to be consistent in the project and remove this part for now.